### ISSUE TITLE
Config screen won't load

### ISSUE THREAD

**Issue by dfxinc** [hjdhjd/homebridge-unifi-protect#1085]
### Homebridge UniFi Protect Version

7.0.0

### Homebridge Platform and OS

Raspberry Pi 5

### Homebridge Version

1.8.2

### Node Version

v20.12.2

### UniFi OS Version

4.0.3

### UniFi Protect Controller Version

4.0.21

### Describe the problem

When selecting "Configure" the screen flashes very briefly the setup screen then displays an image of Homebridge Logo and a Protect camera. There is no option to input data to setup the config...no input is allowed on the setup screen
<img width="815" alt="Screenshot 2024-06-04 at 9 16 02 PM" src="https://github.com/hjdhjd/homebridge-unifi-protect/assets/126509738/29250f3b-1d81-4b91-86b1-fc5ca036e94d">

### Homebridge HBUP JSON configuration

```json
not sure where this is located
```

### Relevant log output

```shell
[04/06/2024, 21:09:33] Loaded plugin: homebridge-unifi-protect@7.0.0
[04/06/2024, 21:09:33] Registering platform 'homebridge-unifi-protect.UniFi Protect'
[04/06/2024, 21:09:33] ---
[04/06/2024, 21:09:33] Loading 2 platforms...
[04/06/2024, 21:09:33] [UniFi Protect] Initializing UniFi Protect platform...
[04/06/2024, 21:09:33] [UniFi Protect] Initializing child bridge 0E:DE:C4:5E:5A:85
Setup Payload:
X-HM://0023OYLHZ9N8T
Enter this code with your HomeKit app on your iOS device to pair with Homebridge:
                       
    ┌────────────┐     
    │ 134-85-399 │     
    └────────────┘     
                       
[04/06/2024, 21:09:33] Homebridge v1.8.2 (HAP v0.12.1) (Homebridge 2A92) is running on port 51932.
[04/06/2024, 21:09:33] [UniFi Protect] Launched child bridge with PID 9485
[04/06/2024, 21:09:33] Registering platform 'homebridge-unifi-protect.UniFi Protect'
[04/06/2024, 21:09:33] [UniFi Protect] Loaded homebridge-unifi-protect v7.0.0 child bridge successfully
[04/06/2024, 21:09:33] [UniFi Protect] No UniFi Protect controllers have been configured.
[04/06/2024, 21:09:33] Homebridge v1.8.2 (HAP v0.12.1) (UniFi Protect) is running on port 57916.
[04/06/2024, 21:15:16] [Homebridge UI] Changes to config.json saved.
[04/06/2024, 21:15:16] [Homebridge UI] Running Command: npm uninstall --save homebridge-unifi-protect
[04/06/2024, 21:15:18] [Homebridge UI] Homebridge restart request received
[04/06/2024, 21:15:18] [Homebridge UI] UI / Bridge settings have not changed; only restarting Homebridge process
[04/06/2024, 21:15:18] [Homebridge UI] Sending SIGTERM to Homebridge
[04/06/2024, 21:15:18] Got SIGTERM, shutting down Homebridge...
[04/06/2024, 21:15:18] Got SIGTERM, shutting down child bridge process...
[04/06/2024, 21:15:23] [HB Supervisor] Homebridge Process Ended. Code: 143, Signal: null
[04/06/2024, 21:15:28] [HB Supervisor] Restarting Homebridge...
[04/06/2024, 21:15:28] [HB Supervisor] Starting Homebridge with extra flags: -I -P /var/lib/homebridge/node_modules --strict-plugin-resolution
[04/06/2024, 21:15:28] [HB Supervisor] Started Homebridge v1.8.2 with PID: 9818
[04/06/2024, 21:15:29] Loaded config.json with 0 accessories and 1 platforms.
[04/06/2024, 21:15:29] Loaded 0 cached accessories from cachedAccessories.
[04/06/2024, 21:15:29] ---
[04/06/2024, 21:15:29] No plugins found.
[04/06/2024, 21:15:29] Loading 1 platforms...
Setup Payload:
X-HM://0023OYLHZ9N8T
Enter this code with your HomeKit app on your iOS device to pair with Homebridge:
                       
    ┌────────────┐     
    │ 134-85-399 │     
    └────────────┘     
                       
[04/06/2024, 21:15:29] Homebridge v1.8.2 (HAP v0.12.1) (Homebridge 2A92) is running on port 51932.
[04/06/2024, 21:15:48] [Homebridge UI] Running Command: /opt/homebridge/lib/node_modules/homebridge-config-ui-x/plugin-upgrade-install.sh homebridge-unifi-protect 7.0.0 /var/lib/homebridge/node_modules
```

### Acknowledgment that you are only running UniFi OS and UniFi Protect releases from the Ubiquiti Official release channel

- [X] I acknowledge that I am not running any early access / beta firmwares or operating systems from either Ubiquiti or Apple.

**Comment by hjdhjd** [comment]
Clear out your browser cache. 😄 Or alternatively load in a private browser window / incognito.

**Comment by dfxinc** [comment]
Thanks for quick response...and thanks for a great plugin!
I installed the previous version and it worked...then updated to 7.0

**Comment by hjdhjd** [comment]
Addressed in v7.0.1. Thanks!
